nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -100,6 +111,61 @@ public void shouldFailWhenMovingToColdStorageDocumentWithoutContent() {
         }
     }
 
+    @Test
+    public void shouldRequestRetrievalDocumentBlobFromColdStorage() {
+        DocumentModel documentModel = createDocument(true);
+
+        // move the blob to cold storage
+        ColdStorageHelper.moveContentToColdStorage(session, documentModel.getRef());
+
+        // request a retrieval from the cold storage content
```

Comment:
"request a retrieval from the cold storage"
or
"request cold storage content retrieval"